src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorPickerSlider.cs

```diff
@@ -172,11 +173,8 @@ private void OnValueChangedEvent(object sender, RangeBaseValueChangedEventArgs a
 			if (m_toolTip is ToolTip toolTip)
 			{
 				toolTip.Content = GetToolTipString();
-
-				// ToolTip doesn't currently provide any way to re-run its placement logic if its placement target moves,
-				// so toggling IsEnabled induces it to do that without incurring any visual glitches.
-				toolTip.IsEnabled = false;
-				toolTip.IsEnabled = true;
```

Comment:
This code matches WinUI and shouldn't be changed. I believe @jeromelaban was already working on the proper fix.

Reply:
I fixed 2 issues( #13868 and #13865) in this PR. Which issue was @jeromelaban working on?

Reply:
@jeromelaban , @Youssef1313 .
No need to PR for issues #13868 and #13865?

Reply:
Indeed, we cannot change that code as the original behavior from winui needs to be kept and tooltip needs to get the behavior that ColorPickerSlider is depending on. I worked on this during last week's stream, see https://www.youtube.com/watch?v=8Txc13MMJQk, though I did not get to finish it.

Reply:
@DierkDroth not at this time, but it can be reused to update this PR.

Reply:
Ah, thanks @jeromelaban for letting us know.